src/Xhgui/Db/PdoWrapper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Xhgui\Db;
+
+use PDO;
+use PDOException;
+
+class PdoWrapper extends PDO
+{
+    /** @var string */
+    private $quoteIdentifier;
+
+    public static function create(array $options): self
+    {
+        $pdoOptions = [
+            PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
+        ];
+        $pdo = new self(
+            $options['dsn'],
+            $options['user'],
+            $options['pass'],
+            $pdoOptions
+        );
+
+        $pdo->quoteIdentifier = self::getQuoteIdentifier($options['dsn']);
```

Comment:
Doesn't https://www.php.net/manual/en/pdo.quote.php help you achieve the same thing?
>  PDO::quote() places quotes around the input string (if required) and escapes special characters within the input string, **using a quoting style appropriate to the underlying driver.**

Reply:
@lauripiisang, unfortunately, no, `quote()` is for values quoting (`"a\""`), but here need to quote identifiers (table names, column names).

in mysql identifiers are quoted with backticks and values with backslash. other systems use double quotes for identifiers and double quotes for an enoding double quote inside double quote separated  value: `""""` is a double quote. actually mysql supports this encode style too.

Reply:
so to cut story short: can't reliably quote column "get" in these pdo drivers:
- pgsql
- mysql
- sqlite

Reply:
sorry, missed that it was about quoting identifiers . yeah, that's unfortunate separate story.

Reply:
According to https://stackoverflow.com/questions/214309/do-different-databases-use-different-name-quote/214344#214344 
>The standard SQL language uses double-quotes for delimited identifiers

> MySQL uses back-quotes by default. MySQL can use standard double-quotes:
```sql
SELECT * FROM `my table`;
SET SQL_MODE=ANSI_QUOTES;
SELECT * FROM "my table";
```

Maybe it's possible to drop backticks in favor of "ansi quotes" ?

Reply:
Something like this
```php
<?php 
        $pdoOptions = [
            PDO::ATTR_ERRMODE => PDO::ERRMODE_EXCEPTION,
        ];
        $adapter = explode(':', $dsn, 2)[0];

        if($adapter === 'mysql'){
            $pdoOptions[PDO::MYSQL_ATTR_INIT_COMMAND] =  'SET SQL_MODE=ANSI_QUOTES;';
        }
        $pdo = new self(
            $options['dsn'],
            $options['user'],
            $options['pass'],
            $pdoOptions
        );

Reply:
that's a good find!

also need to quote the column named `GET`, seems it suits there as well, not selecting a string:

```
15:03:49 mysql{1}> SET SQL_MODE=ANSI_QUOTES;
Query OK, 0 rows affected, 1 warning (0.03 sec)

Warning (Code 3090): Changing sql mode 'NO_AUTO_CREATE_USER' is deprecated. It will be removed in a future release.
15:03:50 mysql{2}> select "GET";
ERROR 1054 (42S22): Unknown column 'GET' in 'field list'
15:03:55 mysql{3}>
```

Reply:
good thing is that it works as low as mysql 5.0:

```

15:06:00 mysql@192.168.40.12:1> SET SQL_MODE=ANSI_QUOTES;
Query OK, 0 rows affected (0.00 sec)

15:06:06 mysql@192.168.40.12:2> select "GET";
ERROR 1054 (42S22): Unknown column 'GET' in 'field list'
15:06:11 mysql@192.168.40.12:3> select @@version;
+------------+
| @@version  |
+------------+
| 5.0.96-log |
+------------+
1 row in set (0.00 sec)

```

Reply:
https://github.com/perftools/xhgui/pull/336

altho maybe acceptable solution would had also been to rename "GET" column to to something else, and leave upgrade notitice that the column must be renamed when upgrading.

Reply:
yeah, I suppose that would be easier and less-hacky. reserved words probably should be avoided.

Reply:
the thing is that reserved new reserved words can appear.

stupid solution to that is to prefix all columns and tables with xhgui :)